src/System.Net.Requests/src/System/Net/HttpDateParse.cs

```diff
@@ -398,5 +398,19 @@ public static bool ParseHttpDate(string dateString, out DateTime result)
 
             return true;
         }
+
+        // parse String to DateTime format.
+        public static DateTime StringToDate(String S)
```

Comment:
"String S" -> "string s"

Reply:
I moved it without modification but will change